wagtailio/core/choices.py

```diff
@@ -46,8 +46,7 @@ class SVGIcon(models.TextChoices):
     ROLLBACK = "rollback", "Rollback"
     SLACK = "slack", "Slack"
     SPEECH_BUBBLE = "speech-bubble", "Speech Bubble"
-    SUN_CLOUD_DARK = "sun-cloud-dark", "Sun Cloud Dark"
-    SUN_CLOUD_LIGHT = "sun-cloud-light", "Sun Cloud Light"
+    SUN_CLOUD = "sun-cloud", "Sun Cloud"
```

Comment:
@engineervix Just FYI i've updated the list of icons after we removed the light/dark versions and added some that were missing